### ISSUE TITLE
Gracefully degrade if VRF is not enabled

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#3717]
If the kernel doesn't have VRF enabled, we can't add routes based on `fwmark` we used to prevent routing loops.

VRF is disabled in WSL for IPv6 and potentially other places, too.

We should gracefully fallback:

- Print a WARNING if VRF is not enabled for IPv4
- Print a WARNING if VRF is not enabled for IPv6
- Still add the route anyway

**Comment by jamilbk** [comment]
I believe this is done / not relevant anymore @conectado ?

**Comment by conectado** [comment]
> I believe this is done / not relevant anymore @conectado ?

no, this is still a problem, but the only distro where we discovered that vrf is disabled until now is WSL and I think no one in the team is using it.

Though it could happen in other distros or for a custom-compiled kernel
